pages/docs/[...slug].tsx

```diff
@@ -107,7 +115,17 @@ export const getStaticProps: GetStaticProps = async function (props) {
   const slug = slugs.join('/')
 
   try {
-    return await getDocProps(props, slug)
+    const results = await client.queries.doc({ relativePath: `${slug}.mdx` })
+    const tinaDocsNavigation = await client.queries.getAllDocs()
+    //TOOD: get rid of line 121. https://github.com/tinacms/tina.io/issues/1982
+    const oldNavDocs = await getDocsNav()
+    return {
+      props: {
+        new: { results },
+        allDocs: { tinaDocsNavigation },
+        oldNavDocs,
+      },
+    }
```

Comment:
Lets get rid of all the old stuff, we dont need it.
If something is using it, change it to use the new stuff

Reply:
Read https://github.com/tinacms/tina.io/issues/1982
duh